docs/vault/guide.md

```diff
@@ -78,6 +215,56 @@ You can decide if your Vault should accept new issue requests by users.
 1. Go to polkadot.js.org/apps -> extrinsics -> VaultRegistry -> acceptNewIssues
 2. Set the collateral and wrapped currency pair (e.g., KSM/KBTC) and if you want to accept new requests (e.g., Yes or No)
 
+## Bitcoin Balance Check
+
+If you don't use the full [monitoring](#monitoring) described below, you should still check once in a while if your Vault client has equal or more BTC in its Bitcoin wallet than the amount of BTC locked on the Interlay or Kintsugi chains. You can compare the two as follows:
+
+1. Navigate to your Vault dashboard and check the "Locked BTC". This gives you the locked BTC that the Interlay or Kintsugi chain assume the Vault holds. In the example below, the `DOT/IBTC` Vault has `7.85948385` BTC locked.
+![BTC_locked](../_assets/img/vault/btc-locked.png)
+2. Connect to the system running the Bitcoin node for your Vault client. You can list the BTC balance of the `DOT/IBTC` Vault by running the following command on the CLI with the keyname you specified when running the Vault client (from the [installation instructions](vault/installation?id=_5-start-the-vault-client)):
+
+```sh
+bitcoin-cli -rpcwallet=<INSERT_YOUR_KEYNAME, example: 0x0e5aabe5ff862d66bcba0912bf1b3d4364df0eeec0a8137704e2c16259486a71>-DOT-IBTC getbalance
+```
+
+The output will look like
+
+```sh
+$ bitcoin-cli -rpcwallet=0x0e5aabe5ff862d66bcba0912bf1b3d4364df0eeec0a8137704e2c16259486a71-DOT-IBTC getbalance
+> 7.8678
+```
+
+In the example given here, the Vault client is healthy: the Vault client has `7.8678` BTC in its Bitcoin wallet and the Interlay chain assumes that it has `7.85948385` BTC locked. That means, even if the Vault is requested to return all BTC, it will have enough BTC to fulfill those requests.
+
+?> If you are unsure about the Bitcoin wallets or you used an alternative for specifiying the Vault keyfile, you can run `bitcoin-cli listwallets` to see all your Vault wallets. You can then insert each item of the returned rpc wallets to check their balance.
```

Comment:
Just fyi, `listwallets` only lists _loaded_ wallets. To get _all_ wallets, you need to use `listwalletdir`

Reply:
Good point, updating